### PR TITLE
[WIP] GED / folder import

### PR DESCRIPTION
**Pull request by pvilalta**

### CODE REVIEW

File: src/i18n/lang/fr.json [hunk 1]
```diff
@@ -227,7 +227,7 @@
     "searchInputPlaceholder": "Rechercher",
     "createModelNotification": "Modèle créé avec succès",
     "structureImport": "Importer une structure GED",
-    "fileImport": "Importer un dossier",
+    "folderImport": "Import un dossier",
```
Comment (NicolasRichel) [inline src/i18n/lang/fr.json]: There is a typo here: "*Import*" instead of "*Importer*"

File: src/utils/projects.js [hunk 2]
```diff
@@ -51,4 +149,11 @@ function treeIdGenerator(projectToImport) {
...
 }
 
-export { projectStatus, treeIdGenerator };
+export {
+  projectStatus,
+  getPaths,
+  getDocsInfos,
+  createTreeFromPaths,
+  matchFoldersAndDocs,
+  treeIdGenerator
```
Comment (NicolasRichel) [inline src/utils/projects.js]: I think these new functions should be in the **files** utils instead of project utils.

**Review by NicolasRichel — CHANGES_REQUESTED** [review]

**Review by Amoki — CHANGES_REQUESTED** [review]
image
After a folder upload, the folder stays in the upload mode and I can't use it.

File: src/components/specific/files/file-upload-card/FileUploadCard.vue [hunk 3]
```diff
@@ -120,8 +148,19 @@ export default {
         }
       },
       onUploadComplete: ({ response: document }) => {
-        uploading.value = false;
-        emit("upload-completed", document);
+        progress.onGoingUploadCounter--;
```
Comment (Amoki) [inline src/components/specific/files/file-upload-card/FileUploadCard.vue]:
```suggestion
        progress.onGoingUploadsCounter--;
```

**Review by Amoki — CHANGES_REQUESTED** [review]

File: src/utils/upload.js [hunk 4]
```diff
@@ -82,4 +89,28 @@ function createFileUploader(
...
+  }
+
+  attrsList.forEach(prop => {
+    input[prop] = attrs[prop];
+  });
+
+  input.addEventListener("change", onChange);
+
+  document.body.appendChild(input);
+  input.click();
```
Comment (Amoki) [inline src/utils/upload.js:120]: Is the input cleared somewhere or it stays in the dom?

**Review by Amoki — CHANGES_REQUESTED** [review]

**Review by Amoki — APPROVED** [review]

**Pull request merged**